app/views/conversations/_show.haml

```diff
@@ -25,5 +25,4 @@
   - for participant in conversation.participants
     = person_image_link(participant, :size => :thumb_small)
 
-.stream
-  = render partial: 'messages', locals: { conversation: conversation }
+= render partial: 'messages', locals: { conversation: conversation }
```

Comment:
Hmm, now do I just change these coding style problems in this single file or all .haml ones?
This PR was actually supposed to be a simple fix, not a rewrite of coding standards 😁